assembl/static2/js/app/components/login/sendPwdForm.jsx

```diff
@@ -16,11 +16,10 @@ class SendPwdForm extends React.Component {
   }
   componentWillReceiveProps(nextProps) {
     const { auth } = nextProps;
-    if (auth.passwordChangeRequest && auth.passwordChangeRequest.success === false) {
-      let msg;
-      if (auth.passwordChangeRequest.data === 404) {
-        msg = I18n.t('login.emailNotFound');
-      } else { msg = I18n.t('login.passwordChangeRequestError'); }
+    const resp = auth.passwordChangeRequest;
+    if (resp && resp.success === false) {
```

Comment:
Ahh, as in there is an absence of a `resp` object? If so, then none of the code below should be run. This is a possible case, as an action can be triggered. However, will a user ever see this case? 99% no, because these actions occur in different components and cannot be triggered in another other way.